app/assets/stylesheets/styleguide/_typography.scss

```diff
@@ -1,218 +0,0 @@
-@import "_print.scss";
-
```

Comment:
`_print` sets:

```
@mixin print {
  @if $is-print {
    @content;
  }
}
```

The only use of `@include print` was in `styleguide/_typography`. If something had been using it I would have expected to see a compile error from sass.